src/index.md

```diff
@@ -9,300 +9,212 @@ description: >
-    If you don't have any mobile development experience, Flutter
-    is an easy and fast way to build beautiful mobile apps.
-    If you are an experienced iOS or Android developer,
-    you can use Flutter for your views and leverage much of your
-    existing Java/Kotlin/ObjC/Swift investment.
-    </p>
-
-    <div class="feature-lists">
-
-        <div class="feature-list-group">
-            <h3>Build</h3>
-
-            <h4>Beautiful app UIs</h4>
-
-                <ul>
-                    <li>Rich 2D GPU-accelerated APIs</li>
-                    <li>Reactive framework</li>
-                    <li>Animation/motion APIs</li>
-                    <li>Material Components and Cupertino widgets</li>
-                </ul>
-
-            <h4>Fluid coding experience</h4>
-
-            <ul>
-                <li>Sub-second, stateful hot reload</li>
-                <li>Refactor, code completion, etc.</li>
-                <li>Dart language and core libs</li>
-                <li>Package manager</li>
-            </ul>
-
-            <h4>Full-features apps</h4>
-
-            <ul>
-                <li>Interop with mobile OS APIs &amp; SDKs</li>
-                <li>Gradle: Java/Kotlin</li>
-                <li>CocoaPods: ObjC/Swift</li>
-            </ul>
+<section class="homepage__community card ">
+    <div class="card-body">
+        <div class="row">
+            <div class="col-md-9 order-1">
+                <!-- TODO(fguerrero): Add map for this section -->
+            </div>
+            <div class="col-md-3">
+                <p><small class="text-muted">Community</small></p>
+                <h3>Find other developers building with Flutter</h3>
+                
+                <input type="number" class="form-control" placeholder="Enter Zip Code" aria-label="Zip Code">
+                <a href="#" class="btn btn-primary">Get Started</a>
+            </div>
         </div>
+    </div>
+</section>
 
-        <div class="feature-list-group">
-            <h3>Optimize</h3>
-
-            <h4>Test</h4>
-
-            <ul>
-                <li>Unit testing</li>
-                <li>Integration testing</li>
-                <li>On-device testing</li>
-            </ul>
-
-            <h4>Debug</h4>
-
-            <ul>
-                <li>IDE debugger</li>
-                <li>Web-based debugger</li>
-                <li>async/await aware</li>
-                <li>Expression evaluator</li>
-            </ul>
-
-            <h4>Profile</h4>
-
-            <ul>
-                <li>Timeline</li>
-                <li>CPU and memory</li>
-                <li>In-app perf charts</li>
-            </ul>
+<div class="homepage__news card-deck">
+    <div class="card">
+        <img class="card-img-top" src="" alt="A news item image">
+        <div class="card-body">
+            <p class="card-text"><small class="text-muted">News</small></p>
+            <h3>Why You Should Start Looking At Google's Flutter And Fuchsia Now</h3>
```

Comment:
?!?!?! Delete "and fuchsia".

Reply:
Also, should this just be in sentence case?

Reply:
I think that was an example headline, not a real news item. I am not sure if this would be like the top 2 blog posts or something like that.